app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -17,17 +18,22 @@ class Api::V2::ContentViewVersionsController < Api::V2::ApiController
     param :version, String, :desc => N_("Filter versions by version number"), :required => false
     param :composite_version_id, :identifier, :desc => N_("Filter versions that are components in the specified composite version"), :required => false
     param_group :search, Api::V2::ApiController
+
```

Comment:
Can remove - we typically have the documentation not separated from the function definition